docs/src/routes/docs/components/file-dropzone.svx

```diff
@@ -51,6 +51,7 @@ For a simpler file input without the built-in representation of the file selecti
 | **`files`** | `[]` | `Array` | The user's selection as an array of [`File`](https://developer.mozilla.org/en-US/docs/Web/API/File) objects. Note: this is an actual `Array`, not a `FileList`, as opposed to the native `<input type="file">`. |
 | **`beforeChange`** | `null` | `Function` | A callback to call for each file that passes the ==`accept`== check before it is added to the ==`files`==. <br /> If it returns a Promise, they will be started for every file, but awaited at the end. |
 | **`disabled`** | `false` | `boolean` | Whether the input should accept files. |
+| **`max`** | `Infinity` | `number` | The maximum number of allowed files. |
```

Comment:
I don't really like the `allowed` here. Maybe `The maximum amount of files that one can upload`?